kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/VaultTestKmsFacade.java

```diff
@@ -221,10 +245,10 @@ else if (response.statusCode() != 200) {
         }
     }
 
-    private void sendRequestExpectingNoContentResponse(HttpRequest request) {
+    private void sendRequestExpectingNoContentResponse(HttpRequest request, int expected) {
```

Comment:
Part of me wonder if an overload of `sendRequestExpectingNoContentResponse` to add a status code might have been better. Then again this triggers people to think about what code the call actually returns.